src/message.ts

```diff
@@ -42,6 +46,46 @@ export class Message {
         }
     }
 
+    /**
+     * Returns the message payload deserialized.
+    */
+    getDataDeserialized(): Uint8Array {
```

Comment:
It needs to be any since data deserialization for 'graphql' returns string.
https://github.com/Big-Vi/memphis.js/blob/feature/consumer-deserializiation/src/message.ts#L77
In this PR, it's converting protobuf to js object. What does pure protobuf mean?